net/wireguard/src/opnsense/scripts/Wireguard/wg-service-control.php

```diff
@@ -222,6 +227,14 @@ function get_stat_hash($fhandle)
                         wg_stop($node);
                         wg_start($node, $statHandle, $carp_if_flag);
                         break;
+                    case 'reload':
+                        if (
+                            @md5_file($node->cnfFilename) != @file_get_contents($node->statFilename) &&
+                            does_interface_exist((string)$node->interface)
+                        ) {
+                            wg_syncconf($node);
+                        }
```

Comment:
```suggestion
                        wg_syncconf($node);
```

This shouldn't check the configuration state as syncconf only touches what is not configured yet... same as 'vpn' hook. Also I wouldn't call this 'reload' even though that matches terminology used by FreeBSD rc(8).... "sync" would be more appropriate.

Reply:
Done in cd906c5eb583bac0ae3bbaf82d4c08fd2f64fc17